src/formats/appledos.textpb

```diff
@@ -0,0 +1,24 @@
+comment: 'Applies standard AppleDOS soft sector skew to the Apple II format'
+is_extension: true
+
+sector_mapping {
+	trackdata {
+		mapping { physical: 0  logical: 0  }
```

Comment:
This is either correct, or exactly backwards.

Reply:
Precisely.